src/packet/header/arranging_header.rs

```diff
@@ -0,0 +1,99 @@
+use super::{HeaderReader, HeaderWriter};
+use crate::error::Result;
+use crate::net::constants::ARRANGING_PACKET_HEADER;
+use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+use std::io::Cursor;
+
+#[derive(Copy, Clone, Debug)]
+/// This header represents a fragmented packet header.
+pub struct ArrangingHeader {
+    arranging_id: u16,
```

Comment:
yeah I definitely think some type aliases are in order

Reply:
Right, let me do that for you.

Reply:
done